make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -214,7 +214,8 @@
                     <arg value="-jar"/>
                     <arg value="${tools.dir}/compiler.jar"/>
                     <arg line="${javac.compilerargs}"/>
-                    <arg value="-g"/>
+                    <arg value="-g:lines"/>
+                    <arg value="--enable-preview"/>
```

Comment:
I suppose the `--enable-preview` trick works because frgaal doesn't emmit `x.5` bytecode versions?

Reply:
Right. Emitted bytecode has no restrictions.